numpy/_core/fromnumeric.py

```diff
@@ -3588,24 +3588,42 @@ def std(a, axis=None, dtype=None, out=None, ddof=0, keepdims=np._NoValue, *,
 
     Notes
     -----
-    The standard deviation is the square root of the average of the squared
-    deviations from the mean, i.e., ``std = sqrt(mean(x))``, where
-    ``x = abs(a - a.mean())**2``.
-
-    The average squared deviation is typically calculated as ``x.sum() / N``,
-    where ``N = len(x)``. If, however, `ddof` is specified, the divisor
-    ``N - ddof`` is used instead. In standard statistical practice, ``ddof=1``
-    provides an unbiased estimator of the variance of the infinite population.
-    ``ddof=0`` provides a maximum likelihood estimate of the variance for
-    normally distributed variables. The standard deviation computed in this
-    function is the square root of the estimated variance, so even with
-    ``ddof=1``, it will not be an unbiased estimate of the standard deviation
-    per se.
+    The definition of the standard deviation of an array is not unique.
+    Assuming the input `a` is a one dimensional NumPy array and `mean` is
+    either provided as an argument or computed as ``a.mean()``, NumPy
+    computes the standard deviation of an array as::
+
+        N = len(a)
+        d2 = abs(a - mean)**2
+        var = d2.sum() / (N - ddof)  # note use of `ddof`
+        std = var**0.5
+
+    Different values of the argument `ddof` are useful in different
+    contexts. NumPy's default ``ddof=0`` corresponds with the expression:
+
+    .. math::
+
+        \sqrt{\frac{\sum_i{|a_i - \bar{a}|^2 }}{N}}
+
+    and is sometimes (for better or for worse) called "population
```

Comment:
You do have some commentary here!  But I honestly think you should be explicit in explaining that a) this is the common term in the field of statistics in particular, and that b) it's based on the distinction between sample and population, and 'population' is in the sense where the values `a` are considered to be the whole population.   Is the term "uncorrected" common in fact?  Robert objected to that, I think on reasonable grounds.

Reply:
Thanks for catching that. I included that only for the reviewers and planned to remove it.

I know there was an objection to "uncorrected", and the rationale that the name shouldn't be used is reasonable. I included the word anyway because it it is a matter of opinion whether it should be used. However, it *is* used, and the purpose of this PR was to connect with what users already know rather than be prescriptive.

It is popular enough that it is the terminology I  use. I don't use "population", yet I included it first because it appears to be more common.